.github/workflows/ansible-test.yml

```diff
@@ -5,7 +5,9 @@
 # 3. For the comprehensive list of the inputs supported by the
 #    ansible-community/ansible-test-gh-action GitHub Action, see
 #    https://github.com/marketplace/actions/ansible-test.
-# 4. If you need help please ask in #ansible-community on the Libera.chat IRC
+# 4. Make sure to add the "check" job to your repository branch
+#    protection once this workflow is added.
```

Comment:
```suggestion
# 4. If you want to prevent merging PRs that do not pass all tests,
#    make sure to add the "check" job to your repository branch
#    protection once this workflow is added.
```

Reply:
Thanks! I think I'll improve this further by saying that a more fine-grained job voting status control is possible too.